recipe/bld.bat

```diff
@@ -0,0 +1,46 @@
+:: TODO: maybe split this into another package.
+set DATADIR="%LIBRAY_PREFIX%/share/coast"
+if not exist %DATADIR% mkdir %DATADIR%
+
+:: GSHHG (coastlines, rivers, and political boundaries):
+rem set EXT="tar.gz"
+rem set GSHHG="gshhg-gmt-2.3.6"
+rem set URL="ftp://ftp.iag.usp.br/pub/gmt/%GSHHG%.%EXT%"
+rem curl %URL% > %GSHHG%.%EXT%
+rem tar xzf %GSHHG%.%EXT%
+rem cp %GSHHG%/* %DATADIR%
+
+:: DCW (country polygons):
+rem DCW="dcw-gmt-1.1.2"
+rem URL="ftp://ftp.soest.hawaii.edu/dcw/%DCW%.%EXT%"
+rem curl %URL% > %DCW%.%EXT%
+rem tar xzf %DCW%.%EXT%
+rem cp %DCW%/* %DATADIR%
```

Comment:
The data download is commented out at the moment. I need to try these commands first to adapt them for Windows.